examples/sveltekit/vite.config.ts

```diff
@@ -0,0 +1,44 @@
+// @ts-nocheck
```

Comment:
I don't like adding this, but for some odd reason I get this error on my `config` object: `Excessive stack depth comparing types '{ plugins: (Plugin_2[] | Plugin_2[])[]; }' and 'UserConfig'.ts(2321)` - I don't know how to debug and assume maybe it's because I'm using unocss via linking instead of installing. I don't know, but I don't think it hurts the purpose of the examples.

Reply:
It's because multiple Vite installed and the type is not aligned.

Reply:
Ok, I understand and see how you fixed it by using defineConfig(). 

Sorry for all the capitalization, spacing, and semicolon oddities. I'm used to using ESLint + Prettier and am still figuring out how your repo functions. Thanks for the cleanup and merge!